jobs/release.Jenkinsfile

```diff
@@ -106,6 +107,21 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
         def gcp_image = ""
         def ostree_prod_refs = [:]
 
+        // arbitrarily selecting the s390x builder; we don't run this
+        // locally because podman wants user namespacing (yes, even just
+        // to push a manifest...)
+        pipeutils.withPodmanRemoteArchBuilder(arch: 's390x') {
```

Comment:
I know why we are using s390x here, but I do worry a little bit about there being some unintentional side effect of this.